src/resources/packages/classy/types/StoreSelectors.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Ticket } from './Ticket';
+
+export type StoreSelectors = {
```

Comment:
As discussed, I think it's fine to have a type for the `Store.d.ts` that contains all store-related types.